packages-exp/firebase-exp/compat/index.ts

```diff
@@ -39,15 +39,15 @@ import 'firebase/<PACKAGE>';
 import firebase from './app';
 import { name, version } from '../package.json';
 
-// import './auth';
+import './auth';
 // import './database';
 // import './firestore';
```

Comment:
firestore/storage being added in other PRs?

Reply:
Good catch. added them here.